client.go

```diff
@@ -75,6 +76,14 @@ func BearerAuth(token string) ClientParam {
 	}
 }
 
+// CookieJar sets http cookie jar for this client
```

Comment:
You have http as `HTTP` elsewhere. And being an acronym, capitalized variant is the way to go